uint8 id_agent
uint8 id_box
float32 distance